governance/delegates/meta/delegates.json

```diff
@@ -0,0 +1,680 @@
+        "externalProfileURL": null
+      },
+      "image": "/home/runner/work/community/community/governance/delegates/0x2C511D932C5a6fE4071262D49bfc018cfBaAa1F5/avatar.jpeg",
```

Comment:
this is supposed to be an IPFS hash right? Does it show this instead because the IPFS image upload failed?

Reply:
you are right, we should control this

Reply:
So with your update, it will just not include an image if it fails to upload?